tsl/test/expected/remote_txn_resolve.out

```diff
@@ -137,6 +137,146 @@ SELECT count(*) FROM _timescaledb_catalog.remote_txn;
      1
 (1 row)
 
+SELECT create_records();
+ create_records 
+----------------
+ 
+(1 row)
+
+--inject errors in the GID and test "commit" resolution for it
+SET timescaledb.debug_inject_gid_error TO 'commit';
+--heal should error out and the prepared transaction should still be visible
+SELECT _timescaledb_internal.remote_txn_heal_data_node((SELECT OID FROM pg_foreign_server WHERE srvname = 'loopback2'));
+WARNING:  "COMMIT PREPARED" failed. Will be retried later
+ remote_txn_heal_data_node 
+---------------------------
+                         0
+(1 row)
+
+SELECT * FROM table_modified_by_txns;
+           describes            
+--------------------------------
+ committed
+ prepared not comitted
+ committed with concurrent heal
+ committed
+(4 rows)
+
+SELECT count(*) FROM pg_prepared_xacts;
```

Comment:
Would be more interesting to se the prepared transactions on the data node(s).

Reply:
Well, that output is not going to be the same everytime we run the test, right?